common/scripted_modifiers/wc_scheme_scripted_modifiers.txt

```diff
@@ -0,0 +1,54 @@
+﻿scheme_base_success_chance_modifier = { #Use MULTIPLIER if you want harsher/softer penalties
```

Comment:
Why you pass `MULTIPLIER`? It's much clearer to pass a value instead of `-35`.
Instead of `add = -15`, it will be:
```
add = {
	value = $value$
	multiply = 0.5
}
```